### PR TITLE
Create journal-week-1.md

### PR DESCRIPTION
**Pull request by JessyRiordan**

### CODE REVIEW

**Comment by pope410211** [comment]
Looks good, very dear diary style ( I think) informative and a bit tl;dr 

:+1:

**Comment by JessyRiordan** [comment]
Thanks @pope410211 and @jorgehjr84  for the awesome feedback. :+1:

File: journal-week-1.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Week One: 5/4 - 5/8
+
+I am only going to touch on the highlights of the week! :smile:
+
+## git status, git add, git commit, git push (always forget this one!) :smile:
+
+### Monday, May 4th
+
+Today was the first day at _The Iron Yard_. Oh lord, what have I gotten myself into! :smile: The topic today: class structure and reviewing homework from pre-work. Everyday we will begin with a stand up meeting. We will all have to state our name, a bright bulb, a dim bulb, and a blocker. The bright bulb will be something that you totally understood, that makes sense for now! The dim bulb is something that you think you did right, but you do not understand how you did it. The blocker is something that you just did not understand, that you could not complete! We took today to orient ourselves with terminal and GitHub. We also began researching the growth vs. fixed mindset. During the reserach I realized that I have to have a growth mindset towards this entire _Iron Yard_ course.
```
Comment (al-the-x) [inline journal-week-1.md]: As an example:
* "mock up" instead of "moch up"

File: journal-week-1.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+### Wednesday, May 6th
+
+Today began with our stand up meeting. The blocker for me was the JavaScript definitions. I am trying to completely understand the concept before I write the definition. This is a weakness of mine that I need to correct! :smile: Moving onto the lesson, today we learned about User Stories. The basic format for a User Story is: "A **someone** can do **something** so that **someone** can get **some** benefit". The smurf (easy) way to remember the format is: "A **smurf** can **smurf** so that **smurfs** can **smurf**". Before you write a User Stories for what you want to creat you must first brainstorm about what that "thing" needs to do. We were brainstorming about a todo app. What does a todo app need to do? This todo app should: add something, check the something off, list all the somethings, delete the somethings, prioritize something, and edit something. Once you have created your User Stories from your brainstorming list you can begin to develop some code that can help you create the todo app.
```
Comment (al-the-x) [inline journal-week-1.md]: > Moving onto the lesson, today we learned about User Stories. The basic format for a User Story is:

Maybe condense this into one sentence?

**Pull request merged**